homeassistant/components/logbook.py

```diff
@@ -239,6 +259,38 @@ def humanify(events):
                     entity_id)
 
 
+def _exclude_events(events, config):
+    """Get lists of excluded entities and platforms."""
+    excluded_entities = []
+    excluded_domains = []
+    exclude = config[DOMAIN].get(CONF_EXCLUDE)
+    if exclude:
+        excluded_entities = exclude.get(CONF_ENTITIES, [])
```

Comment:
You've already specified a default value in `CONFIG_SCHEMA`, so no need to specify one here. You can just use `exclude[CONF_ENTITIES]`